enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/components/Summarize.tsx

```diff
@@ -50,6 +50,8 @@ const SummarizeInner = ({
   };
 
   const {
+    query,
```

Comment:
I would argue that the issue is not fully fixed yet.

> Query builder does not reflect what's displayed on screen 

That sentence is still true. "Count" is shown as if it was applied when it isn't.
I don't see any reason why would we have this one special case.
It's confusing from user's perspective and from code perspective.

----

I see 2 valid ways for it to work.
When you click "Summarize" (for a question without aggregations in the last stage), either:
1. "Count" aggregation is actually applied to the question (without having to click "Done"). Personally I don't like this, but maybe there's a good product reason for it.
2. Nothing happens, user sees "empty state" in the sidebar. User has to choose aggregations/breakouts manually.
    - I'd change one more thing in this case - show breakouts section regardless of whether aggregations are present. Scenario: 
        1. New question > Orders > join Products > Breakout by whatever > Visualize
        2. Summarize
        3. Remove the _default_ "Count" aggregation
        4. Now we don't show the breakout in the sidebar, but it's there. Confusing.

Reply:
@kamilmielnik this is been discussed multiple times and the existing behavior is what we want. Please take a look https://metaboat.slack.com/archives/C0645JP1W81/p1719588433987439